MultiVersions/Forge/main/enderio/lang/zh_tw.json

```
@@ -0,0 +1,543 @@
+  "gui.enderio.ioconfig.disabled": "已停用",
+  "gui.enderio.ioconfig.neighbour": "顯示／隱藏相鄰方塊",
+  "gui.enderio.ioconfig.none": "停用",
+  "gui.enderio.ioconfig.pull": "輸入端",
+  "gui.enderio.ioconfig.push": "輸出端",
+  "gui.enderio.no_capacitor": "安裝電容器才能使用這台機器。",
+  "gui.enderio.no_power": "沒有足夠的能量來使用這台機器",
+  "gui.enderio.no_soulbound": "將靈魂綁定在物品上來使用它",
+  "gui.enderio.no_source": "蓄水器下方需要水源才能開始運作",
+  "gui.enderio.nocap.desc": "安裝任何電容器\n來使這台機器開始運作！",
+  "gui.enderio.nocap.title": "缺少電容器",
+  "gui.enderio.ok": "Ok",
+  "gui.enderio.output_full": "輸出欄沒有足夠的空間",
+  "gui.enderio.progress": "進度 %s%%",
+  "gui.enderio.range": "運作範圍",
+  "gui.enderio.range.hide": "隱藏運作範圍",
+  "gui.enderio.range.show": "顯示運作範圍",
+  "gui.enderio.redstone.active_with_signal": "接收到紅石訊號時運作",
+  "gui.enderio.redstone.active_without_signal": "沒有紅石訊號時運作",
+  "gui.enderio.redstone.always_active": "總是運作",
+  "gui.enderio.redstone.mode": "紅石模式",
+  "gui.enderio.redstone.never_active": "從不運作",
+  "gui.enderio.redstone_channel": "紅石頻道",
+  "gui.enderio.round_robin.disabled": "輪詢調度：已停用",
+  "gui.enderio.round_robin.enabled": "輪詢調度：已啟用",
+  "gui.enderio.sag_mill_chance": "機率：%s%%",
+  "gui.enderio.sag_mill_chance_ball": "機率：%s%%（已受研磨珠影響）",
+  "gui.enderio.self_feed.disabled": "自我供給：已停用",
+  "gui.enderio.self_feed.enabled": "自我供給：已啟用",
+  "gui.enderio.spawner.disabled": "已被設定檔停用",
+  "gui.enderio.spawner.other_mod": "已被其他模組停用",
+  "gui.enderio.spawner.too_many_mob": "附近太多生物",
+  "gui.enderio.spawner.too_many_spawner": "附近太多生怪磚",
+  "gui.enderio.spawner.unknown": "未知生物",
+  "gui.enderio.visible.false": "隱藏",
+  "gui.enderio.visible.true": "可見",
+  "guidebook.enderio.book_title": "書本標題",
+  "guidebook.enderio.landing_text": "著陸頁",
+  "hint.enderio.connected_textures.text": "若想讓電容庫帶有連接紋理，你可能會想在用戶端上安裝《Athena》模組",
+  "info.enderio.coordinate_selector.no_block": "範圍內沒有方塊",
+  "info.enderio.coordinate_selector.no_paper": "物品欄中沒有紙",
+  "info.enderio.energy.amount": "%s µI",
+  "info.enderio.too_many_levels": "你的等級已超過 21862 級，經驗值太多了。",
+  "item.enderio.advanced_item_filter": "進階物品篩選器",
+  "item.enderio.animal_token": "生物標記",
+  "item.enderio.basic_capacitor": "基礎電容器",
+  "item.enderio.basic_fluid_filter": "基礎流體篩選器",
+  "item.enderio.basic_item_filter": "基礎物品篩選器",
+  "item.enderio.black_paper": "黑紙",
+  "item.enderio.broken_spawner": "損壞的生怪磚",
+  "item.enderio.cake_base": "蛋糕基底",
+  "item.enderio.chemical_conduit": "化學導管",
+  "item.enderio.clayed_glowstone": "黏土螢石粉",
+  "item.enderio.cloud_seed_bucket": "雲之精華桶",
+  "item.enderio.cloud_seed_concentrated_bucket": "濃縮雲之精華桶",
+  "item.enderio.cold_fire_igniter": "寒焰打火石",
+  "item.enderio.conductive_alloy_grinding_ball": "導電合金研磨珠",
+  "item.enderio.conductive_alloy_ingot": "導電合金錠",
+  "item.enderio.conductive_alloy_nugget": "導電合金粒",
+  "item.enderio.conduit_binder": "導管黏合器",
+  "item.enderio.conduit_binder_composite": "導管黏合器複合材料",
+  "item.enderio.confusing_powder": "混亂粉",
+  "item.enderio.coordinate_selector": "	座標選擇儀",
```

這似乎是全形空格，PackSquash 在報錯
看你要一格空格還兩格空格
```suggestion
  "item.enderio.coordinate_selector": " 座標選擇儀",
```